site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -94,23 +97,33 @@
         //set the right panel to allow adding new students
         add_students_div.append("Add New Students:<br />")
         let student_autofill = JSON.parse($('#student_autofill').val());
-        add_students_div.append('<p class="add_user_ids" id="add_user_id_0"><input type="text" name="add_user_id_0" /><br></p>');
+        add_students_div.append('<input class="add_user_ids" type="text" name="add_user_id_0" /><br>');
         $('[name="add_user_id_0"]', form).autocomplete({
             source: student_autofill
         });
         $('[name="add_user_id_0"]').autocomplete( "option", "appendTo", form );
-        add_students_div.append('<p id="add_student_button" style="cursor: pointer;" onclick="addNewStudent();" aria-label="Add More Users"><i class="fas fa-plus-square"></i>Add More Users<br></p>');        
+        add_students_div.append('<p id="add_student_button" style="cursor: pointer;" onclick="addStudentTextbox(\'#admin-gradeable-peers-form\',\'#students-to-add\');" aria-label="Add More Users"><i class="fas fa-plus-square"></i>Add More Users<br></p>');  
     }
 
-    function checkSubmitValid() {
+    function checkNewStudentsValid(grader_id, user_input, valid_students, new_grader) {
```

Comment:
Could you add a JS Doc comment block to this function to explain its purpose and the variable types for longer JS functions its harder to figure what its doing at brief and what variable types it needs.

E.g:

```js
/**
* example description here
*
* @param {String} grader_id the id of grader to blah
* @param {String} user_input 
* @param {Array} valid_students list of foo
*/

https://jsdoc.app/tags-param.html
```